specification/v4.0/OSDM-online-api-v4.0.0-draft.yml

```diff
@@ -7685,10 +7699,7 @@ components:
         format:
           $ref: '#/components/schemas/DocumentFormat'
           nullable: true
```

Comment:
Both content and format must be nullable false now. We mad the whole DocumentContent optional.